flytectl/pkg/ext/fetcher.go

```diff
@@ -55,7 +55,7 @@ type AdminFetcherExtInterface interface {
 	FetchAllVerOfWorkflow(ctx context.Context, name, project, domain string, filter filters.Filters) ([]*admin.Workflow, error)
 
 	// FetchWorkflowLatestVersion fetches latest version of workflow in a  project, domain
-	FetchWorkflowLatestVersion(ctx context.Context, name, project, domain string, filter filters.Filters) (*admin.Workflow, error)
+	FetchWorkflowLatestVersion(ctx context.Context, name, project, domain string) (*admin.Workflow, error)
```

Comment:
Should we keep the filter here? What if I want to fetch the latest workflow with a specific tag?

Reply:
i think better to remove it actually, it's clearer in what it's doing, and merging filters behind the scenes seems weird to me.